python/tvm/driver/tvmc/model.py

```diff
@@ -337,7 +385,21 @@ def import_package(self, package_path: str):
         t = tarfile.open(package_path)
         t.extractall(temp.relpath("."))
 
-        if os.path.exists(temp.relpath("metadata.json")):
+        if self.use_vm:
```

Comment:
Instead of requiring users to remember if a saved package uses the VM or not, maybe we can combine the VM handling with the classic format handling and set `self.use_vm` if we find `lib.tar` and `lib.so` instead of `mod.tar` and `mod.so`.